pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -667,18 +667,20 @@ func (dm *DockerManager) runContainer(
 	}
 
 	// Set sysctls if requested
-	sysctls, unsafeSysctls, err := api.SysctlsFromPodAnnotations(pod.Annotations)
-	if err != nil {
-		dm.recorder.Eventf(ref, api.EventTypeWarning, events.FailedToCreateContainer, "Failed to create docker container %q of pod %q with error: %v", container.Name, format.Pod(pod), err)
-		return kubecontainer.ContainerID{}, err
-	}
-	if len(sysctls)+len(unsafeSysctls) > 0 {
-		hc.Sysctls = make(map[string]string, len(sysctls)+len(unsafeSysctls))
-		for _, c := range sysctls {
-			hc.Sysctls[c.Name] = c.Value
+	if container.Name == PodInfraContainerName {
```

Comment:
I wonder whether the issue really an issue: on kubelet restart all pods go through pod admission again. I would expect that pods now invalid in 1.4 would be terminated. Is the behavior supposed to be like that?

Strangely, during my manual 1.3->1.4 testing I don't see that. Still investigating.